lte/gateway/python/magma/pipelined/ebpf/ebpf_manager.py

```diff
@@ -85,8 +98,9 @@ def get_ebpf_manager(config):
 class ebpf_manager:
     def __init__(self, sgi_if_name: str, s1_if_name: str, gw_ip: IPAddress, bpf_ul_file: str = BPF_UL_FILE, bpf_dl_file: str = BPF_DL_FILE):
         self.enabled = True
-        self.b_ul = BPF(src_file=bpf_ul_file, cflags=[''])
-        self.b_dl = BPF(src_file=bpf_dl_file, cflags=[''])
+        with cd(BPF_PATH):
```

Comment:
Lets pass BPF_PATH as config, to configure 'orc8r/gateway/c/common/ebpf/' for unit and integ tests. this would avoid copying the file back and forth during development.

Reply:
Added passing, will add the testing in a new PR because also need to figure out a way to copy the EbpfMap header to the same dir as the test files.